signac/__main__.py

```diff
@@ -805,7 +805,7 @@ def main_config_set(args):
     try:
         cfg = config.read_config_file(fn_config)
     except OSError:
-        cfg = config.get_config(fn_config)
+        cfg = config.read_config_file(fn_config)
```

Comment:
This try/except is no longer meaningful. The same function is being called twice.

Reply:
Interesting... that means the try-except was also previously not meaningful since `get_config` called `read_config_file` internally. This must be a vestige of some earlier version of the code.